pkgs/tools/system/clinfo/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, ocl-icd, opencl-headers }:
+
+stdenv.mkDerivation rec {
+  pname = "clinfo";
+  version = "2.2.18.04.06";
+
+  src = fetchurl {
+    url = "https://github.com/Oblomov/${pname}/archive/${version}.tar.gz";
+    sha256 = "f77021a57b3afcdebc73107e2254b95780026a9df9aa4f8db6aff11c03f0ec6c";
+  };
+
+  buildInputs = [ ocl-icd opencl-headers ];
+
+  NIX_LDFLAGS = [ "-lOpenCL" ];
+
+  installPhase = ''
```

Comment:
`makeFlags = [ "PREFIX=${placeholder "out"}" ];` should allow you to get rid of the custom `installPhase`.

Reply:
You can also just pass PREFIX=$(out) in this case, and Make will expand the environment variable $out.